examples/embedded/src/main/java/org/eclipse/jetty/embedded/ServerWithAnnotations.java

```diff
@@ -61,7 +61,7 @@ public static final void main( String args[] ) throws Exception
         new Transaction(new com.acme.MockUserTransaction());
 
         // Define an env entry with webapp scope.
-        new EnvEntry(webapp, "maxAmount", new Double(100), true);
+        new EnvEntry(webapp, "maxAmount", Double.valueOf(100), true);
```

Comment:
Use just `100.0D` rather than `Double.valueOf(100)`. Same below.

Reply:
Fixed in updated commits.